lib/pbio/platform/ev3/pbsysconfig.h

```diff
@@ -1,17 +1,18 @@
 // SPDX-License-Identifier: MIT
-// Copyright (c) 2020-2024 The Pybricks Authors
+// Copyright (c) 2020-2025 The Pybricks Authors
 
 #define PBSYS_CONFIG_FEATURE_BUILTIN_USER_PROGRAM_REPL             (1)
 #define PBSYS_CONFIG_FEATURE_BUILTIN_USER_PROGRAM_PORT_VIEW        (0)
 #define PBSYS_CONFIG_FEATURE_BUILTIN_USER_PROGRAM_IMU_CALIBRATION  (0)
 #define PBSYS_CONFIG_FEATURE_PROGRAM_FORMAT_MULTI_MPY_V6           (1)
 #define PBSYS_CONFIG_FEATURE_PROGRAM_FORMAT_MULTI_MPY_V6_3_NATIVE  (0)
 #define PBSYS_CONFIG_BATTERY_TEMP_ESTIMATION        (1)
+#define PBSYS_CONFIG_HMI                            (1)
+#define PBSYS_CONFIG_HMI_STOP_BUTTON                (1 << 7) // top left, back button
```

Comment:
Why not using macro names instead of hard-coding numbers?

Reply:
Including `pbio` which includes `pbdrv` here seems to mess things up, leading them to be always zero.